iree/test/microbenchmarks/transpose_benchmarks.mlir

```diff
@@ -0,0 +1,124 @@
+//===----------------------------------------------------------------------===//
```

Comment:
This reminds me that they may not be compiled at some point. We don't set targets to test  microbenchmark/*.mlir compilation...  IIRC, the dot_general microbenchmark is broken atm..

I think we can add compilation for these files. To do it, you can write a test suite in BUILD. And then run `build_tools/bazel_to_cmake/bazel_to_cmake.py` to fix cmake files.

See an example in https://github.com/google/iree/blob/406d7b33c5e7770ef8055e4fd5b1bbf9219baf0b/iree/test/e2e/xla_ops/BUILD#L398

Let's exclude broken files at this moment. (I guess mhlo_conv and mhlo_dot_general have been broken for a while) I'll send out a fix-up for broken files.